lib/matplotlib/widgets.py

```diff
@@ -3443,6 +3465,71 @@ def __init__(self, ax, onselect, useblit=False,
         self.grab_range = grab_range
 
         self.set_visible(True)
+        self._draw_box = draw_box
+        self._box = None
+        self._box_handle_props = self._handle_props.update(box_handle_props) or None
+        self._box_props = box_props or None
+
+    def _get_bbox(self):
+        return self._selection_artist.get_bbox()
+
+    def _add_box(self):
+        self._box = RectangleSelector(self.ax,
+                                      onselect=lambda *args, **kwargs: None,
+                                      useblit=self.useblit,
+                                      grab_range=self.grab_range,
+                                      handle_props=self._box_handle_props,
+                                      props=self._box_props,
+                                      interactive=True)
+        self._box.connect_event('motion_notify_event', self._scale_polygon)
+        self._update_box()
```

Comment:
```suggestion
        self._box._state_modifier_keys.pop('rotate')
        self._box.connect_event('motion_notify_event', self._scale_polygon)
        self._update_box()
```
Currently the box with the default modifiers responds to the `r` key and can be rotated, but of course without effect on the polygon. Until rotation support is added as well, I think this could avoid some confusion.